ui/platform_window/x11/x11_window_manager_ozone.cc

```diff
@@ -10,11 +10,19 @@ namespace ui {
 
 X11WindowManagerOzone::X11WindowManagerOzone() : event_grabber_(None) {}
 
-X11WindowManagerOzone::~X11WindowManagerOzone() {}
+X11WindowManagerOzone::~X11WindowManagerOzone() {
```

Comment:
one suggestion, mainly to keep the diff against upstream shorter:

can we change X11WindowManagerOzone methods to take X11WindowOzone as paramater instead of XID?

Then it could do all this control without need to add X11WindowManagerOzone::Add/Remove/GetX11Window.